tests/testthat/test_TaskSurv.R

```diff
@@ -11,7 +11,10 @@ test_that("right censoring", {
   expect_equal(task$censtype, "right")
   expect_numeric(task$times())
   expect_integer(task$status())
-  expect_equal(task$formula(), as.formula(Surv(time, status, type = "right") ~ .))
+  expect_equal(
+    task$formula(), as.formula(Surv(time, status, type = "right") ~ .),
+    ignore_attr = TRUE
```

Comment:
digged this up a bit - it's for backwards compatibility?

Reply:
its just for ignoring attributes, since here the attributes are different, but in practice expect_equal from v2 didn't check for attributes see the docs: https://testthat.r-lib.org/reference/expect_equivalent.html